HarpGuru/OverlayMenu/MenuOptions/PozitionOption/types/index.ts

```diff
@@ -0,0 +1,3 @@
+import { MenuProps } from '../../../types'
+
+export type PozitionOptionProps = Pick<MenuProps, 'activeHarpStrata' | 'setActiveHarpStrata' | 'activeDisplayMode'>
```

Comment:
Is there a way to get all of the covariance related options to share a type? They will necessarily be the same as one another.

Reply:
Not necessary. This module is clear enough. If we need to enhance it later then we can consider a relevant refactor beforehand to make that easier.